2018---The REAL First Power Up/src/org/usfirst/frc/team2586/robot/Robot.java

```diff
@@ -201,10 +204,10 @@ public void teleopInit() {
 	public void teleopPeriodic() {
 
 		// Get joystick inputs for drive base
-		double driveLeft = leftStick.getY() * -1;
-		double driveRight = rightStick.getY() * -1;
+		double driveLeft = deadZoneComp(leftStick.getY() * -1);
+		double driveRight = deadZoneComp(rightStick.getY() * -1);
 
-		mainDrive.tankDrive(driveLeft, driveRight);
+		teleopGyro(driveLeft, driveRight);
```

Comment:
Use two buttons the be able to disable this and have pure manual control. Mainly for the event that the gyro goes nuts, you want to be able to continue to play the match.

Reply:
done

Reply:
I also added two buttons for turning (turn either direction at 0.3 I believe) without having to use the joysticks for more precise stuff like lining it up.